### PR TITLE
Adapt to ewok changes in passing dates

### PR DESCRIPTION
**Pull request by shlyaeva**
Companion PR for https://github.com/JCSDA-internal/ewok/pull/489

Tested both 3DVar and HofX marine ewok experiments with this branch, first cycle completed for both.

### CODE REVIEW

**Review by ytremolet — APPROVED** [review]
Thanks.

File: ewok/tasks/runSaveAnalysis.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 os.chdir(workdir)
 
 # Date
-andate = conf['date']
+andate = conf['an']['date']
```
Comment (cmgas) [inline ewok/tasks/runSaveAnalysis.py:23]: This was conf['an']['datetime'] in fv3, is the difference normal here?

**Review by cmgas — COMMENTED** [review]
Just a question

File: ewok/tasks/runSaveAnalysis.py (hunk 1 above)
Comment (shlyaeva) [reply ewok/tasks/runSaveAnalysis.py:23]: Good eye! Yes, this is expected. (I started with `datetime` :))
soca specifies date in the analysis yaml section differently from fv3-jedi (this is model-controlled yaml section):
in fv3-jedi `datetime` key is used: https://github.com/JCSDA-internal/fv3-jedi/blob/develop/ewok/templates/analysis.yaml#L7
in soca `date` key is used: https://github.com/JCSDA-internal/soca/blob/89fbf9bf88e7e66e2de9668b1b00ab1c0c4827a7/ewok/templates/analysis.yaml#L2

**Review by cmgas — APPROVED** [review]

**Pull request merged**